src/styles/cards.scss

```diff
@@ -0,0 +1,22 @@
+.main__cards {
```

Comment:
Move to main.css

[checklist p9](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)